plugins/backstage-plugin-flux/src/components/FluxRuntimeCard/FluxRuntimeCard.test.tsx

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import { renderInTestApp, TestApiProvider } from '@backstage/test-utils';
+import { configApiRef } from '@backstage/core-plugin-api';
+import { ConfigReader } from '@backstage/core-app-api';
+import {
+  KubernetesApi,
+  kubernetesApiRef,
+  KubernetesAuthProvidersApi,
+  kubernetesAuthProvidersApiRef,
+} from '@backstage/plugin-kubernetes';
+import { KubernetesRequestBody } from '@backstage/plugin-kubernetes-common';
+import { FluxRuntimeCard } from './FluxRuntimeCard';
+
+const makeTestFluxController = (
+  name: string,
+  namespace: string,
+  images: string[],
+  clusterName: string,
+  labels: { [name: string]: string },
+) => {
+  return {
+    name,
+    namespace,
+    conditions: [
+      {
+        type: 'Available',
+        status: 'True',
+        reason: 'MinimumReplicasAvailable',
+        message: 'Deployment has minimum availability.',
+        timestamp: '',
+      },
+      {
+        type: 'Progressing',
+        status: 'True',
+        reason: 'NewReplicaSetAvailable',
+        message: 'ReplicaSet has successfully progressed.',
+        timestamp: '',
+      },
+    ],
+    images,
+    suspended: false,
+    clusterName,
+    uid: '4527e05c-eed4-489d-93ae-0cd66ca3277e',
+    labels,
+  };
```

Comment:
Ah, structure here is not quite right I think,

```
metadata: { name, namespace, labels },
```

And we can rm `images`, `suspended`, `clusterName`, they're not there on the k8s deployments response.